etc/checkstyle/checkstyle.xml

```diff
@@ -42,7 +42,7 @@
 
     <module name="LineLength">
         <property name="fileExtensions" value="java"/>
-        <property name="max" value="120"/>
+        <property name="max" value="130"/>
```

Comment:
The current value is getting in the way of the test quality above (the last line of the error is the problematic line)

I considered a few workarounds that seem to impact the readability of the test; I am open to suggestions

Reply:
1) try `@SuppressWarnings("checkstyle:LineLength")`
2) alternatively, you can try getting rid of the text block and breaking the lines
but please, let's not adjust the checkstyle for the sake of one line in tests :-)